tests/lib/__init__.py

```diff
@@ -100,6 +100,10 @@ def packages2(self):
     def packages3(self):
         return self.root.join("packages3")
 
+    @property
+    def packages4(self):
```

Comment:
This feels unnecessary -- why has a simple package been added under a new, different directory; am I missing something?